packages/deployment/app/config.yml

```diff
@@ -98,7 +98,8 @@ default:
 
   ecs:
     restartTasksOnDeploy: false
-    amiid: ami-6944c513
+    container_stop_timeout: 2m
+    amiid: ami-007571470797b8ffa
```

Comment:
Don't the DAAC's have to use the NGAP base ami? Can we ask it's built from this base ami?